docs/templates/tutorial.md

```diff
@@ -0,0 +1,85 @@
+# Your first template
+
+A common way to create a template is to begin with a starter template
+then modify it for your needs. Coder makes this easy with starter
+templates for popular development targets like Docker, Kubernetes,
+Azure, and so on. Once your template is up and running, you can edit
+it in the Coder web page. Coder even handles versioning for you so you
+can publish official updates or revert to previous revisions.
+
+In this tutorial, you'll create your first template from the Docker
+starter template.
+
+
+## Before you start
+
+You'll need a computer or cloud computing instance with both
+[Docker](https://docs.docker.com/get-docker/) and
+[Coder](../install/index.md) installed on it.
```

Comment:
If you install Docker after Coder, you need to add the `coder` user to the `docker` group.